resources/user-agents/browsers/android-browser/android-browser-4-0.json

```diff
@@ -810,7 +810,7 @@
             "Vodafone SmartTab II 10": "Lenovo SmartTab II 10",
             "X10": "SonyEricsson X10",
             "XOLO A700": "XOLO A700",
-            "XOOM 2 ME": "Motorola MZ608",
```

Comment:
I changed any of these that didn't have the explicit model name to the generic "Xoom 2" and "Xoom 2 ME" device.